.github/workflows/actions/get-core-dependencies/action.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Get Core Dependencies'
+description: 'initializes core dependencies after repo has been pulled'
+runs:
+  using: composite
+  steps:
+    - name: Use Node 16
+      uses: actions/setup-node@56337c425554a6be30cdef71bf441f15be286854 # v3.1.1
+      with:
+        node-version: 16
+        cache: 'npm'
+
+    - name: NPM Cache (Unix)
```

Comment:
is this step (and the windows-specific step below) necessary? In the test output it looks like a caching operation is happening twice when this job is running:

<img width="1335" alt="Screen Shot 2022-04-20 at 9 50 08 AM" src="https://user-images.githubusercontent.com/6207644/164245754-3aff23b7-9d0a-462e-ace1-35f4758723c4.png">

In the documentation for the `setup-node` action it says that caching the global package cache for npm between builds should be supported by setting `cache: 'npm'` - am I missing something here, or are we possibly duplicating that here?

Reply:
Nope - good call, that shouldn't be needed anymore. I ported the caching of `node_modules/` to caching `~/.npm` and forgot that setup-node does that for us 😝